meson.build

```diff
@@ -117,3 +117,40 @@ gnome.post_install(
     update_mime_database: get_option('update_mime_database'),
     update_desktop_database: true,
 )
+
+fs = import('fs')
+builddir = meson.current_build_dir()
+sourcedir = meson.current_source_dir()
+if host_machine.system() == 'windows'
+  name='gtkwave'
+  name_cap = 'GTKWave'
+
+  arch = 'x86_64-w64-mingw32'
+  host = 'w64'
+
+  nsis_cmd = [
+    find_program('makensis'), 
+    '-DBUILDDIR='+builddir,
+    '-DSOURCEDIR='+sourcedir,
+    '-DICON_DIR=../../compilation/windows',
+    '-DNAME='+name,
+    '-DNAME_CAP='+name_cap,
+    '-DICON_NAME='+name + '_logo',
+    '-DARCH='+arch,
+    '-DSYSROOT=/usr/' + arch + '/sys-root/mingw',
+    '-DHOST='+host,
+    '-DVERSION='+'v'+ meson.project_version(),
+    '-DLIBGCCDLL=libgcc_s_seh-1.dll',
+    '@INPUT@',
+  ]
+
+  windows_installer = custom_target(
+    'windows_installer',
+    output: 'install-gtkwave.exe',
+    input: files('./compilation/windows/gtkwave.nsi'),
+    build_always_stale: true,
+    command: nsis_cmd,
+    depends: [gtkwave_exe]
+  )
+  alias_target('windows_installer', windows_installer)
```

Comment:
This caused problems when I tested it:
```suggestion
```

Reply:
If I don't add line 155, the `ninja -t targets` doesn't show `windows_installer`. How did it cause a problem?